### ISSUE TITLE
PlaceholderImageScaleType doesn't work

### ISSUE THREAD

**Issue by QXH313** [facebook/fresco#2481]
### PlaceholderImageScaleType doesn't work

- When I use the attribute of the roundedcorner， placeholderImageScaleType not normal work, like I set, on the system version for Android P's phone

- Fresco version：com.facebook.fresco:fresco:2.1.0

```
 <com.facebook.drawee.view.SimpleDraweeView
        android:id="@+id/draweePoiIcon"
        android:layout_width="70dp"
        android:layout_height="50dp"
        android:layout_marginStart="12dp"
        android:layout_marginTop="12dp"
        android:scaleType="fitCenter"
        app:actualImageScaleType="fitCenter"
        app:placeholderImage="@drawable/poi_detail_image_place_holder"
        app:placeholderImageScaleType="fitCenter"
        app:roundedCornerRadius="5dp" />

```

- The actual rendering shown：

<img width="903" alt="rendering" src="https://user-images.githubusercontent.com/20204169/77502818-ec7d3600-6e96-11ea-9eff-d8a06d024d3d.png">

- The source file used：
![original_image](https://user-images.githubusercontent.com/20204169/77502893-251d0f80-6e97-11ea-925c-6a944f9b4148.png)

- I hope to get your help. Thank you very much

@oprisnik

**Comment by oprisnik** [comment]
Please see https://github.com/facebook/fresco/issues/2260 and https://github.com/facebook/fresco/issues/2277

**Comment by lynn-kollectin** [comment]
I use fresco 2.2.0.
I'm still having this issue even after putting the png file to /res/drawable or /res/drawable-nodpi.
But it's normal if the file is vector drawable.

**Comment by oprisnik** [comment]
Is your placeholder a normal PNG image?

**Comment by lynn-kollectin** [comment]
It looks normal. And it works fine when I used Glide.

**Comment by oprisnik** [comment]
What image format? A simple PNG?

**Comment by oprisnik** [comment]
Does it work if you set the placeholder image in code but don't specify the resource ID but the Drawable itself when the Drawable is loaded via `Drawable placeholder = ContextCompat.getDrawable(context, R.drawable.name)`? See https://developer.android.com/reference/androidx/core/content/ContextCompat#getDrawable(android.content.Context,%20int)

**Comment by QXH313** [comment]
I've solved this problem by moving the image resource into the drawable-nodpi file
@lynn-kollectin @oprisnik

**Comment by lynn-kollectin** [comment]
Thanks @oprisnik.
If I set placeholder image programmatically, and I remove fresco:placeholderImage from xml, it works ok no matter I use Drawable or resource ID.
If I didn't remove fresco:placeholderImage from xml, it still displays abnormally.
(My those png files are in drawable-nodpi now)

**Comment by oprisnik** [comment]
Yeah, I think the issue is that we do not use ContextCompat to load the Drawable but the system's version. We should change this. Pull requests are welcome :)

**Comment by liyadongcn** [comment]
i have the same problem that the placeholderimagescaletype does not work. it came out from the 1.10.0 version. now i use the 1.9.0 version it works well. In the XML file it is fresco:placeholderImage ="@drawable/umeng_socialize_wechat",the drawble is a png file.

**Comment by Abhi-y2003** [comment]
Hey, I would like to work on this.
